docs/source/using/addons.adoc

```diff
@@ -161,6 +161,11 @@ sleep::
 If the add-on command starts with `sleep`, it waits for the specified number of seconds.
 This can be useful in cases where you know that a command such as `oc` might take a few seconds before a certain resource can be queried.
 
+cat::
+If the add-on command starts with `cat`, the arguments following the `cat` command should be a valid file path.
+This will print the file content to the console if valid file path otherwise error message about non-exist file.
```

Comment:
`This will print the file content to the console on valid file path else displays the error message about non-existence of file.`